tensorrt_llm/_torch/auto_deploy/utils/sharding_utils.py

```
@@ -110,48 +171,286 @@ def split_tensor(
+        rank: Current rank
+        world_size: Total number of ranks
+        add_dist: Whether to add distribution op after entry_node
+        min_local_shape: Minimum local shape constraint
+        weights_to_shard: Optional list of regex patterns to match weight names
+        weight_shard_dims: Optional dict mapping weight keys to their shard dimensions
+        fused_weight_dims: Optional dict mapping weight keys to their fused dimension lists
+        quantization_cb: Optional quantization callback
+    """
+    # Find next linear node to define subgraph boundary
+    try:
+        next_lin_node, depth = bfs(entry_node, is_any_lin_op, include_root=False)
```

Ruff (0.14.2)

Unpacked variable `depth` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)